src/vireshJJun20/Assignment17/StringMembers.java

```diff
@@ -0,0 +1,41 @@
+package vireshJJun20.Assignment17;
+
+public class StringMembers {
+
+	void CheckString(String str){
+		int cLetter=0;
+		int cLower=0;
+		int cUpper=0;
+		int cDigit=0;
+		int cOther=0;
+		for (int i=0; i<str.length(); i++){
+			if(Character.isLetter(str.charAt(i))){
+				cLetter++;
```

Comment:
Are you. sure this if block is required. ? cLetter without using this variable you can print expected output.

Reply:
I got your point !! (I added that if block to explicitely include isLetter() method.. ) 
Thanks Krishna. 
Updated my code and now it look like below..
![image](https://user-images.githubusercontent.com/66830247/85831992-e1df7f00-b7ac-11ea-84cc-dc0a48e9e812.png)

Reply:
Now it's perfect code.